lib/inspect_rpmdeps.c

```diff
@@ -260,6 +263,15 @@ static bool check_explicit_lib_deps(struct rpminspect *ri, Header h, deprule_lis
                 continue;
             }
 
+            /*
+             * issue 1246: Prevent false positives when the library is provided
+             * by a subpackage coming from another SRPM.
+             */
+            peer_srpm = headerGetString(peer->after_hdr, RPMTAG_SOURCERPM);
```

Comment:
Blank line after line 270, please.

Reply:
I also moved the comment immediately above the condition check, it makes more sense.